packages/table/src/_docs.scss

```diff
@@ -128,17 +128,54 @@ Loading states
 
 When fetching or updating data, it may be desirable to show a loading state. The Table components
 provide fine-grain loading control of loading row headers, column headers, or individual cells.
+Several table components expose a `loading` or `loadingOptions` prop, but loading-related rendering
+is computed with components lower in the hierarchy taking priority. For example, a cell with
+`loading` set to `false` will never render its loading state even if the `Column` component to which
+it belongs has a `loadingOptions` value of `[ ColumnLoadingOption.CELLS ]`. The following examples
+display a table of the largest potentially hazardous asteroid (based on absolute magnitude)
+discovered in a given year.
+
+Weight: 4
+
+Styleguide components.table-js.loading
+*/
+
+/*
+Table
+
+`Table` expose a `loadingOptions` prop that allows you to control the loading state behavior of all
+column header, row header, and body cells. Try toggling the different options.
+
+@react-example TableLoadingExample
+
+Styleguide components.table-js.loading.table
+*/
+
+/*
+Columns
+
+`Column` also expose a `loadingOptions` prop that allows you to control the loading state behavior
+of an individual columns' header cell and body cells. Try selecting a different column.
```

Comment:
oh i get it, there's a `<select>` to choose columns. but that sentence sure sounds like you want me to click on a column to select it (ie, make it blue). clarify the wording, that you're simply asking folks to play with the option.